src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -282,7 +290,10 @@ def displayImageAtIndices(self, indices: ImageIndices):
         with h5py.File(self._file_name, "r") as h5file:
             nxprocess = h5file[self._nxprocess_path]
             map_shape = get_dataset(nxprocess, "result/intensity").shape
-            image_index = row * map_shape[1] + col + self._offset
+            if self._map_ptr is not None:
```

Comment:
According to above, `self._map_ptr` should never be `None` here, right ?

Reply:
Can you guarantee `initData` is always run before-hand ?

Reply:
Yes: if `initData`, was not run, `self._file_name` will be `None` and the function will return at line 286 just above.